stacks/apps/exchange.yml

```diff
@@ -1147,7 +953,30 @@ Resources:
         - { Key: prx:cloudformation:root-stack-id, Value: !Ref RootStackId }
         - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
         - { Key: prx:dev:application, Value: Exchange }
-      TaskRoleArn: !GetAtt TaskRole.Arn
+  SayWhenScheduledRule:
+    Type: AWS::Events::Rule
+    Condition: EnableWorkers # See README
+    Properties:
+      Description: !Sub Periodically runs ${EnvironmentType} SayWhen jobs
+      ScheduleExpression: rate(1 hour)
+      State: ENABLED
+      Targets:
+        - Arn: !Ref EcsClusterArn
+          EcsParameters:
+            TaskCount: 1
+            TaskDefinitionArn: !Ref WorkerTaskDefinition
```

Comment:
Oh, I see.  Reuse the worker task def, but run a different command.

Reply:
right, I don't think it has any other unique needs